assets/src/app/ato/GettingStarted.tsx

```diff
@@ -11,20 +12,37 @@ const GettingStarted: React.FunctionComponent<any> = (props) => {
               <TextContent>
                   <Text component="h1">Getting Started</Text>
                   <Text component="p">
-                      The Red Hat ATO Pathways microsite provides resources to accelerate your ATO process.
+                    The Red Hat ATO Pathways microsite provides resources to help accelerate your ATO process.
                   </Text>
+                    <Text component="h2">What Exactly is an ATO?</Text>
+                    <Text component="p">
+                    It's your agency authority certifying your system is ready for day 2: system availability, routine maintenance, and monitoring, and lots of other concerns.
```

Comment:
I would probably remove "day 2" and change it to "operations". Reason being is that in some cases it is pre-day 1 or day 1.

Reply:
We are redefining this whole section